### ISSUE TITLE
Miner cannot connect

### ISSUE THREAD

**Issue by userid2018** [Snipa22/nodejs-pool#430]
Hello,

I am using xmr-stak as Miner and have configured as required. but i cannot connect to the pool.

I tried to connect to XMRpool.net:5555 from the miner to test the connection and it worked but it doesn't connect to the pool which i have installed. My Pool is on Ubuntu VM hosted on Azure. Do I need to open any specific port or perform any steps ?

**Comment by shigutso** [comment]
yes you need to forward the mining ports on your router in order to be able to connect and mine

**Comment by userid2018** [comment]
We opened required port in Azure Vm but still cant get it connected. This is a result of netstat run on poolserver

netstat -ltunp
(Not all processes could be identified, non-owned process info
 will not be shown, you would have to be root to see it all.)
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:25325           0.0.0.0:*               LISTEN      -
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      -
tcp        0      0 0.0.0.0:18080           0.0.0.0:*               LISTEN      26223/monerod
tcp        0      0 127.0.0.1:18081         0.0.0.0:*               LISTEN      26223/monerod
tcp        0      0 127.0.0.1:3306          0.0.0.0:*               LISTEN      -
tcp6       0      0 :::80                   :::*                    LISTEN      -
tcp6       0      0 :::22                   :::*                    LISTEN      -

**Comment by bobbieltd** [comment]
Check if your coin daemon (monerod) is fully synced ?

**Comment by bobbieltd** [comment]
@userid2018 Good job ! Syncing without downloading raw blockchain taking 3 days is normal.
No admin page = very good
UPDATE pool.config SET item_value = '4xx put your wallet here' WHERE module = 'pool' and item = 'address';
UPDATE pool.config SET item_value = '4xx put your another wallet here' WHERE module = 'payout' and item = 'feeAddress';
UPDATE pool.config SET item_value = 'later' WHERE module = 'general' and item = 'mailgunKey';
UPDATE pool.config SET item_value = 'later' WHERE module = 'general' and item = 'mailgunURL';
UPDATE pool.config SET item_value = 'admin@mypool.com' WHERE module = 'general' and item = 'emailFrom';
UPDATE pool.config SET item_value = 'http://127.0.0.1:8000/leafApi' WHERE module = 'general' and item = 'shareHost';
pool config.json
bindip : 0.0.0.0
hostname : whatevermypool

**Comment by userid2018** [comment]
@bobbieltd 

I had these entries already but to make sure I ran update again. still I dont see Port 5555 being listened on when i run netstat -tulnp

**Comment by userid2018** [comment]
@bobbieltd 

just noticed once more things that my monroed is running only  on 18080 . where do we specify this ?

If I am not mistaken this should also be running on port 18081 as well. i do have entry in pool table

14	daemon	port	18081	int	Monero Daemon RPC Port



result of netstat -tulnp
-------------------------


Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:25324           0.0.0.0:*               LISTEN      1929/ruby
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      1386/sshd
tcp        0      0 0.0.0.0:18080           0.0.0.0:*               LISTEN      45780/monerod
tcp        0      0 127.0.0.1:3306          0.0.0.0:*               LISTEN      1602/mysqld
tcp6       0      0 :::80                   :::*                    LISTEN      1346/caddy
tcp6       0      0 :::22                   :::*                    LISTEN      1386/sshd
tcp6       0      0 :::8000                 :::*                    LISTEN      2187/init.js
tcp6       0      0 :::8001                 :::*                    LISTEN      2153/init.js

**Comment by bobbieltd** [comment]
18080 is monerod p2p port
18081 is monerod rpc port
18082 is monero wallet rpc port

Coin daemon always uses two port (one for p2p binding 0.0.0.0 and one for rpc binding 127.0.0.1)
Wallet daemon uses one port (rpc binding 127.0.0.1)

You don’t need to specify 18080, it is default in monero codes.

**Comment by bobbieltd** [comment]
Check if monerod is synced with
tail -f ~/.bitmonero/bitmonero.log
